presto-test-coverage/pom.xml

```diff
@@ -0,0 +1,615 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.facebook.presto</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>0.272-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-test-coverage</artifactId>
+    <name>presto-test-coverage</name>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+
+        <maven.deploy.skip>true</maven.deploy.skip>
+        <air.check.skip-enforcer>true</air.check.skip-enforcer>
+        <air.check.skip-enforcer>true</air.check.skip-enforcer>
+        <air.check.skip-duplicate-finder>true</air.check.skip-duplicate-finder>
+        <air.check.skip-findbugs>true</air.check.skip-findbugs>
+        <air.check.skip-dependency>true</air.check.skip-dependency>
+    </properties>
+
+    <dependencies>
+        <dependency>
```

Comment:
high-level question: is there an automatic way to include a new module? For example, if someone adds a new module `presto-my-new-module-1` and forgets to add it here, how can we actively prevent such situation?

Reply:
Definitely a good question. I believe we could make a [custom rule](https://maven.apache.org/enforcer/enforcer-api/writing-a-custom-rule.html) for the maven enforcer plugin. I wouldn't know where the best place to put that code would be. If that's inconvenient then we could also try and enforce dependencies through a github action that lists out modules, and the dependencies of the presto-test-coverage and ensures they're in sync or bypassed by a property in the pom. Those were the first things that come to mind at least.